gobblin-modules/gobblin-http/src/main/java/gobblin/http/ApacheHttpRequest.java

```diff
@@ -0,0 +1,38 @@
+package gobblin.http;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.apache.http.HttpEntityEnclosingRequest;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.util.EntityUtils;
+
+import gobblin.async.AsyncRequest;
+
+
+public class ApacheHttpRequest<D> extends AsyncRequest<D, HttpUriRequest> {
```

Comment:
It's an `AsyncRequest` specific to `HttpUriRequest`. It helps to log the actual request via `toString` method